spec/numberformat.html

```diff
@@ -713,14 +725,35 @@ <h1>Abstract Operations for NumberFormat Objects</h1>
       <h1>
         CurrencyDigits (
           _currency_: a String,
+          _currencyPrecision_: a String,
+        ): a non-negative integer
+      </h1>
+      <dl class="header">
+      </dl>
+      <emu-alg>
+        1. Assert: IsWellFormedCurrencyCode(_currency_) is *true*.
+        1. Assert: _currency_ is equal to the ASCII-uppercase of _currency_.
+        1. If the Common Locale Data Repository supplemental data pertaining to fractional currency values contains an element with _currency_ as the value of its iso4217 attribute, then
+          1. If _currencyPrecision_ is *"cash"* and the element with _currency_ as the value of its iso4217 attribute has a cashDigits attribute, return the value of that attribute; otherwise, return the value of that element's digits attribute.
+        1. Return 2.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-currencyrounding" type="abstract operation">
+      <h1>
+        CurrencyRounding (
```

Comment:
Suggestion: `CurrencyRoundingIncrement` (not clear this returns a rounding increment)